src/components/RichTextEditor/serializer/utils/serializeLeafToHtml.ts

```diff
@@ -11,7 +11,7 @@ import {
 import escapeHtml from 'escape-html';
 
 export const serializeLeafToHtml = (node: TDescendant): string => {
```

Comment:
instead of TDescendant make any

Reply:
and also in file serializeNodeToHtmlRecursive.ts where prop is passed

Reply:
Thanks, just replaced `TDescendant` for `any`